contracts/StartGame.sol

```diff
@@ -17,20 +25,47 @@ contract StartGame is ChainlinkClient, Ownable {
         path = _path;
     }
 
-    function createWordRequest(uint256 payment) internal returns (bytes32) {
+    function requestStartGame(uint256 payment) public returns (bytes32) {
         // newRequest takes a JobID, a callback address, and callback function as input
-        Chainlink.Request memory req = buildChainlinkRequest(CHAINLINK_JOB_ID, this, this.fulfill.selector);
+        Chainlink.Request memory req = buildChainlinkRequest(CHAINLINK_JOB_ID, this, this.fullfillStartGame.selector);
         req.add("url", url);
         req.add("path", path);
-        bytes32 requestId = sendChainlinkRequestTo(chainlinkOracleAddress(), req, payment);
+        bytes32 requestId = sendChainlinkRequest(req, payment);
+        //requestId will point to the player and the game address(which is pending)
+        requestIdToGame[requestId] = Game(msg.sender, address(0));
+        //return request id as to check if we've gotten a game back
         return requestId;
     }
 
-    function fulfill(bytes32 _requestId, bytes32 _data)
+    function fullfillStartGame(bytes32 _requestId, bytes32 _data)
         public
         recordChainlinkFulfillment(_requestId)
         returns (bytes32) {
-        return _data;
+            //get the game instance
+            Game storage gameInstance = requestIdToGame[_requestId];
+
+            require(gameInstance.player != 0, "Id does not exist");
+
+            //create the new hangman with the word(data)
+            Hangman game = new Hangman(bytes32ToBytes(_data), _data.length);
+            //save game into the request
+            gameInstance.game = game;
+            //update the owner of the game
+            game.transferOwnership(gameInstance.player);
+    }
+
+    function bytes32ToBytes(bytes32 data) private pure returns (bytes) {
+        uint i = 0;
+        while (i < 32 && uint(data[i]) != 0) {
+            ++i;
+        }
+        bytes memory result = new bytes(i);
+        i = 0;
+        while (i < 32 && data[i] != 0) {
+            result[i] = data[i];
+            ++i;
+        }
+        return result;
```

Comment:
This brilliant solution is from:
https://ethereum.stackexchange.com/questions/40920/convert-bytes32-to-bytes